src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -290,31 +296,65 @@ public void addClassLiterals(
   }
 
   /**
-   * Given a file containing signatures, returns the operations for them.
+   * Given a file containing fully-qualified method signatures, returns the operations for them.
    *
    * @param file a file that contains method or constructor signatures, one per line
-   * @param onlyMethods if true, throw an exception if a constructor is read
-   * @return a map from each class type to the set of methods/constructors in it
+   * @return a map from each class type to the set of methods and constructors in it
    * @throws OperationParseException if a method signature cannot be parsed
    */
-  public static MultiMap<Type, TypedOperation> readOperations(Path file, boolean onlyMethods)
+  public static MultiMap<Type, TypedClassOperation> readOperations(Path file)
       throws OperationParseException {
-    MultiMap<Type, TypedOperation> sideEffectFreeMap = new MultiMap<>();
     if (file != null) {
       try (EntryReader er = new EntryReader(file, "(//|#).*$", null)) {
-        for (String line : er) {
-          String sig = line.trim();
-          TypedClassOperation operation =
-              signatureToOperation(
-                  sig, VisibilityPredicate.IS_ANY, new EverythingAllowedPredicate());
-          sideEffectFreeMap.add(operation.getDeclaringType(), operation);
-        }
+        return OperationModel.readOperations(er);
+
       } catch (IOException e) {
         String message = String.format("Error while reading file %s: %s%n", file, e.getMessage());
         throw new RandoopUsageError(message, e);
```

Comment:
Could this be a bug in Randoop, if it is reading a built-in file?

Reply:
Now checked in caller in readSideEffectFreeMethods().